cupy/logic/comparison.py

```diff
@@ -38,6 +38,28 @@
 )
 
 
+def array_equal(a, b):
```

Comment:
For compatibility to `numpy.array_equal`
```suggestion
def array_equal(a1, a2):
```